src/test/java/org/elasticsearch/search/preference/SearchPreferenceTests.java

```diff
@@ -42,24 +48,24 @@
 public class SearchPreferenceTests extends ElasticsearchIntegrationTest {
 
     @Test
-    public void testThatAllPreferencesAreParsedToValid(){
+    public void testThatAllPreferencesAreParsedToValid() {
         //list of all enums and their strings as reference
-        assertThat(Preference.parse("_shards"),equalTo(Preference.SHARDS));
-        assertThat(Preference.parse("_prefer_node"),equalTo(Preference.PREFER_NODE));
-        assertThat(Preference.parse("_local"),equalTo(Preference.LOCAL));
-        assertThat(Preference.parse("_primary"),equalTo(Preference.PRIMARY));
-        assertThat(Preference.parse("_primary_first"),equalTo(Preference.PRIMARY_FIRST));
-        assertThat(Preference.parse("_only_local"),equalTo(Preference.ONLY_LOCAL));
-        assertThat(Preference.parse("_only_node"),equalTo(Preference.ONLY_NODE));
+        assertThat(Preference.parse("_shards"), equalTo(Preference.SHARDS));
+        assertThat(Preference.parse("_prefer_node"), equalTo(Preference.PREFER_NODE));
+        assertThat(Preference.parse("_local"), equalTo(Preference.LOCAL));
+        assertThat(Preference.parse("_primary"), equalTo(Preference.PRIMARY));
+        assertThat(Preference.parse("_primary_first"), equalTo(Preference.PRIMARY_FIRST));
+        assertThat(Preference.parse("_only_local"), equalTo(Preference.ONLY_LOCAL));
+        assertThat(Preference.parse("_only_node"), equalTo(Preference.ONLY_NODE));
```

Comment:
Would it be okay if we removed the whitespace only changes from this pull request?